roles/confluent.variables_handlers/defaults/main.yml

```diff
@@ -54,15 +55,18 @@ ssl_self_signed_ca_cert_filename: snakeoil-ca-1.crt
 ssl_self_signed_ca_key_filename: snakeoil-ca-1.key
 ssl_self_signed_ca_password: capassword123
 
+# this one not working with c3
+# ssl_cipher_suites: TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+ssl_cipher_suites: TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+
 # Zookeeper Variables
 zookeeper_user: "{{zookeeper_default_user}}"
 zookeeper_group: "{{zookeeper_default_group}}"
 zookeeper_sasl_protocol: "{{sasl_protocol if sasl_protocol == 'kerberos' else 'none'}}"
 # Variable only used by Kafka for the time being, health checks/components will use 2181
 zookeeper_client_port: "{{'2182' if zookeeper_ssl_enabled|bool else '2181'}}"
-# TODO set these equal to ssl_enabled and ssl_mutual_auth_enabled on next version increase
-zookeeper_ssl_enabled: false
-zookeeper_ssl_mutual_auth_enabled: false
+zookeeper_ssl_enabled: "{{ssl_enabled}}"
```

Comment:
this will most likely break some ssl scenarios, so there should be changes elsewhere